### DISCUSSION TITLE
Custom sizes

### DISCUSSION THREAD

**Discussion by AdamOndrejkovic** [tailwindlabs/tailwindcss#8268] [Help]
Hello I am working on a project and would like to add a specific size for example h-[1200px] but when I try to do so I get an error: 
`The `h-[1200px]` class does not exist. If you're sure that `h-[1200px]` exists, make sure that any `@import` statements are being properly processed before Tailwind CSS sees your CSS, as `@apply` can only be used for classes in the same CSS tree.`

**Comment by carlos-dubon** [comment]
for this to work you need to enable `JIT` mode in your `tailwind.config.js` file:

```
{ 
    mode: 'jit',
    purge: ['./pages/**/*.{js,ts,jsx,tsx}', './components/**/*.{js,ts,jsx,tsx}'],
    darkMode: false, // or 'media' or 'class'
    theme: {
        extend: {},
    },
    variants: {
        extend: {},
    },
    plugins: [],
}
```
